### PR TITLE
Streamline zipcode edit UI in search results

### PR DESCRIPTION
**Pull request by ElvisJG**
https://user-images.githubusercontent.com/44843903/182963424-ae0ea7b0-b4f9-4a69-848c-be6450e29525.mp4

Autofocus, style change, order change, mobile css mediaquery to change order and flex-basis

### CODE REVIEW

**Review by amdarnell7 — CHANGES_REQUESTED** [review]
My recommendation would be to get ride of the blue highlight box when clicking on change but I would defer to @admoorgit if there is a more effective way to approach the visual

**Comment by ElvisJG** [comment]
> My recommendation would be to get ride of the blue highlight box when clicking on change but I would defer to @admoorgit if there is a more effective way to approach the visual

That is the focus box from the browser, it's needed to meet a11y standards and 508 compliance

File: src/components/ZipInput.tsx [hunk 1]
```diff
@@ -6,21 +6,27 @@ type ZipInputProps = {
...
   showError = false,
 }: PropsWithChildren<ZipInputProps>) {
   const { t } = useTranslation();
 
   return (
     <div className="width-full">
-      <Label htmlFor="zip" className="margin-bottom-1">
-        {t(`common.zipCode`)}
-      </Label>
+      {!noLabel && (
```
Comment (jlhogan) [inline src/components/ZipInput.tsx:25]: I think label is required for input components, accessibility-wise. to hide you should add the class “usa-sr-only” (or maybe it’s just “sr-only” but I think it has USA prefix) which will make the label only ‘visible’ to screen readers but not in the actual visible DOM.

**Review by admoorgit — CHANGES_REQUESTED** [review]
Works and looks as anticipated. Yay! To address @amdarnell7's comment — 
@ElvisJG I wonder if we can just add a lil margin (.25rem) to the right of the search field so the button has some breathing room between it and the focus stroke.

Or could we put the focus stroke inside the box? Down with either solution.

File: src/components/ZipInput.tsx (hunk 1 above)
Comment (ElvisJG) [reply src/components/ZipInput.tsx]: great call, done 💯

**Comment by ElvisJG** [comment]
> Works and looks as anticipated. Yay! To address @amdarnell7's comment —  @ElvisJG I wonder if we can just add a lil margin (.25rem) to the right of the search field so the button has some breathing room between it and the focus stroke.
> 
> Or could we put the focus stroke inside the box? Down with either solution.

Done 😀

**Review by admoorgit — APPROVED** [review]
Looks good to me!

**Review by jlhogan — APPROVED** [review]

**Comment by jlhogan** [comment]
@amdarnell7 this is ready for another pass by you!

**Review by amdarnell7 — APPROVED** [review]
This looks great thank you!

**Pull request merged**